dashboard/app/controllers/transfers_controller.rb

```diff
@@ -4,71 +4,51 @@ class TransfersController < ApplicationController
 
   # POST /sections/:id/transfers
   def create
-    # TODO(asher): Much of the permissioning here should be done through CanCan.
     new_section_code = params[:new_section_code]
+    current_section_code = params[:current_section_code]
+    unless new_section_code && current_section_code
+      return head :bad_request
```

Comment:
Nope. Given that the UI should always be submitting these parameters, I'm interested in data validation but not error transparency.